.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,50 @@
+<!-- When opening a PR, start by forking this repository. Then, based on the type of change you're making you'll need to create a new branch from either the `master` or `live` branches:
+
+For documentation for new features, please base your fork off the master branch.
+
+If you have a typo or existing document improvement to an already shipped feature, please base your change off of the [live branch](https://github.com/MicrosoftDocs/WindowsCommunityToolkitDocs/tree/live).  This will allow us to get the change to the published documentation between releases.
+
+We will periodically merge updates from the live branch to master to keep master in-sync with the published docs.  When we make a new release, we will push master to the live branch in order to publish documentation for new features.
+
+Documentation Links
+- [Staging review from 'master' branch](https://review.docs.microsoft.com/en-us/windows/uwpcommunitytoolkit/?branch=master)
+- [Live site from 'live' branch](https://docs.microsoft.com/en-us/windows/uwpcommunitytoolkit) -->
+
+## Fixes # <!-- Link to relevant issue (for ex: #1234) which will create reference to the associated issue once the PR is created. -->
```

Comment:
We should call out here to that we should also link to the feature PR in the Windows Community Toolkit repo